### PR TITLE
Refactor handling version string

### PR DESCRIPTION
**Pull request by shadowwalkersb**
This is to make `regro-cf-autotick-bot` happy next time it needs to update the feedstock.

### CODE REVIEW

**Comment by jakirkham** [comment]
So this PR should be from a fork as it would normally release as soon as CIs pass. That said, for this change, it probably doesn't matter since a new release shouldn't be made by these adjustments.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {% set name = "libgit2" %}
-{% set version_major   = "0" %}
-{% set version_minor   = "26" %}
-{% set version_revison = "0" %}
-{% set version = ".".join([version_major, version_minor, version_revison]) %}
+{% set version = "0.26.0" %}
 {% set sha256 = "6a62393e0ceb37d02fe0d5707713f504e7acac9006ef33da1e88960bd78b6eac" %}
+
+{% set version_major   = version.split(".")[0] %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Is `version_major` used?

**Comment by jakirkham** [comment]
One question, otherwise LGTM. :)

File: recipe/meta.yaml (hunk 1 above)
Comment (shadowwalkersb) [reply recipe/meta.yaml]: Nope. And how did you catch that?!

**Comment by shadowwalkersb** [comment]
> So this PR should be from a fork as it would normally release as soon as CIs pass. That said, for this change, it probably doesn't matter since a new release shouldn't be made by these adjustments.

Figured since there is no version or build number bumping, it would run the tests, but there would be no uploads.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Eagle eyes. 🦅

**Comment by jakirkham** [comment]
Sure, again it's not a problem for this PR. Just a friendly note. :)

**Review by jakirkham — APPROVED** [review]

**Comment by shadowwalkersb** [comment]
Any suggestions how to fix Appveyor failures?

**Comment by shadowwalkersb** [comment]
> Any suggestions how to fix Appveyor failures?

Restarting Appveyor jobs fixed it. Merging.

**Comment by jakirkham** [comment]
Sorry. Missed the question (and the failures). There was a release of `conda-verify` that we ended up pulling.

**Pull request merged**